src/main/java/com/codessquad/qna/service/QuestionService.java

```diff
@@ -52,5 +61,54 @@ private void checkPermission(HttpSession session, Question question) {
             throw new IllegalUserAccessException();
         }
     }
+
+    private class PageNumber {
+        private int pageId;
+        private boolean active;
+        private int pageNumber;
```

Comment:
```suggestion
        private int pageId;
        private int pageNumber;
        private boolean active;
```

필드 순서도 관련도가 높은 것 끼리 묶어주시는 것을 추천합니다.